src/ui/actions/session.ts

```diff
@@ -62,6 +61,8 @@ export function getAccessibleRecording(
         if (expectedError) {
           dispatch(setExpectedError(expectedError));
           return null;
+        } else {
+          dispatch(selectors.clearExpectedError());
```

Comment:
nit: Can we just import `clearExpectedError` (without the `selectors` wrapper):
```suggestion
          dispatch(clearExpectedError());
```
Not a fan of the `import * as blah` pattern we use in the codebase. I think it mostly exists for class components, to distinguish between the "connected" action and the imported one, but for function components this distinction is unnecessary 😄 

As for the "suspect" part – I guess you're right, since other things might call `setExpectedError`

Is there some way we could check the current error, and only clear it if it's the exact type this module sets?

Reply:
`tokenManager` is definitely up to be revisited sometime soon. It's messy at best.

I've added a new helper action to clear just the recording access errors which I think makes it more clear and safer.